packages/utils/jest.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    rootDir: './',
+    globals: {
+        'ts-jest': {
+            tsconfig: 'tsconfig.lib.json',
+        },
+    },
+    moduleFileExtensions: ['js', 'ts'],
+    testMatch: ['**/*.test.ts'],
+    coverageDirectory: './coverage/',
+    collectCoverage: true,
+    collectCoverageFrom: ['**/src/**/*.ts'],
+    modulePathIgnorePatterns: ['node_modules'],
+    // setupFiles: ['./tests/jest.setup.js'],
```

Comment:
please remove, if we will need it we can copy paste it later

Reply:
:+1: